### PR TITLE
Unify styling of delete menu items further.

### PR DESCRIPTION
**Pull request by linuspahl**
## Description

Usually we define a red text color for a delete menu item in menus.

image

There were some cases were we did not apply this styling.

image

I started with adding `variant="danger"` to these `MenuItem`s, but ended up creating a dedicated component `MenuItemDelete` and implemented it for these cases.

/nocl
/prd https://github.com/Graylog2/graylog-plugin-enterprise/pull/9128

### CODE REVIEW

File: graylog2-web-interface/src/components/common/EntityList.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ const items = entities.map(entity => {
                     actions={[
                       <Button key={1} bsStyle="info">Edit</Button>,
                       <DropdownButton key={2} id="more-dropdown" title="More" pullRight>
-                        <MenuItem variant="danger">Delete</MenuItem>
+                        <MenuItem>Edit</MenuItem>
```
Comment (maxiadlovskii) [inline graylog2-web-interface/src/components/common/EntityList.md:40]: I think we have to use here  delete menu item

**Review by maxiadlovskii — COMMENTED** [review]
In the DeleteMenuItem component, it would be nice to have default children prop 'Delete'. In this case would be no need to pass it whenever we use it. 

UPD
Also, we can keep the danger variant in MenuItem and use it in DeleteMenuItem. In the future, we can add more variants to MenuItem. Also, we might need some `danger` action which is not deleting at all. But that has a downside. We might end up in a situation where we have usage of DeleteMenuItem and `<MenuItem variant="danger" danger>Delete</MenuItem>`

File: graylog2-web-interface/src/components/common/EntityList.md (hunk 1 above)
Comment (linuspahl) [reply graylog2-web-interface/src/components/common/EntityList.md:40]: Thanks!

**Comment by linuspahl** [comment]
@maxiadlovskii I like the idea of using `Delete` as default child for `DeleteMenuItem` and adjusted the code.

**Review by maxiadlovskii — APPROVED** [review]

**Pull request merged**